lambda/models/clients/litellm_client.py

```diff
@@ -58,42 +54,38 @@ def add_model(self, model_name: str, litellm_params: Dict[str, str], additional_
         two models with the same name, which causes ambiguous results when using the OpenAI API for listing models as
         that only shows one model per model name.
         """
-        requests.post(
+        resp = requests.post(
             self._base_uri + "/model/new",
-            headers=self._headers,
             json={
                 "model_name": model_name,
                 "litellm_params": litellm_params,
-                "model_info": additional_metadata,
             },
             timeout=self._timeout,
-            verify=self._verify,
         )
+        return resp.json()  # type: ignore [no-any-return]
 
-    def delete_model(self, unique_id: str) -> None:
+    def delete_model(self, identifier: str) -> None:
         """
         Delete a model from the database.
 
-        The unique_id is the ID that LiteLLM generates on its end when creating a model, regardless of if the model
+        The identifier is the ID that LiteLLM generates on its end when creating a model, regardless of if the model
```

Comment:
Is it accurate that LiteLLM generates this?

Reply:
yes